### PR TITLE
Revert ling nerf

### PR DESCRIPTION
**Pull request by ghost**
:cl: steamp0rt
del: Ling nerfs got reverted
del: Horrorform still exists, though
add: As it turns out, Horrorform was actually just OP's mom.
add: Fart sting.
/:cl:

Closes #9464

### CODE REVIEW

**Comment by Manly-Dwarf** [comment]
+1

It's about time we reverted the ling nerfs as a whole. Ling being insanely OP is the only way we're going to successfully get this gamemode removed.

Clearly, nerfing the gamemode was not what I wanted. As a whole, ling critically needs a rework and should not stay in rotation. So fuck it, let's just have it be super OP again and make everyone realize how OP all of lings' buttons are.

**Comment by Manly-Dwarf** [comment]
OH WAIT YOU UNINCLUDED REMOVING THE LINGS' ANTISTUNS
OHHHH YES. WE GOIN STRAIGHT DOWN THE SHITTER THEN. GODMODE AWAY!

File: hippiestation.dme [hunk 1]
```diff
@@ -1178,6 +1178,7 @@
 #include "code\modules\antagonists\changeling\changeling.dm"
 #include "code\modules\antagonists\changeling\changeling_power.dm"
 #include "code\modules\antagonists\changeling\powers\absorb.dm"
+#include "code\modules\antagonists\changeling\powers\adrenaline.dm"
```
Comment (Manly-Dwarf) [inline hippiestation.dme]: HAHAHAHAHAHAHAHA
THIS WASN'T EVEN PART OF MY REMOVAL PR BUT FUCK IT
"CHANGELING'S WHOLE GIMMICK IS BEING OP"
LET'S SHOW EM ALL

File: hippiestation/code/modules/antagonists/changeling/horrorform.dm [hunk 2]
```diff
@@ -22,7 +22,8 @@
 						"<span class='userdanger'>We cast off our petty shell and enter our true form!<br>This form will not last forever, so devour as many people as possible!</span>")
 	var/mob/living/simple_animal/hostile/true_changeling/new_mob = new(get_turf(user))
 	new_mob.real_name = pick("True Form Changeling", "panic stinger", "chaos bringer", "Revelations 11:15-19", "Space Satan", \
-	 "teegee coder", "fun destroyer", "lean sipper", "guy who put pineapple on cornpotato pizza", "oversized ham disc", "greyshirt's bane")
+	 "teegee coder", "fun destroyer", "lean sipper", "guy who put pineapple on cornpotato pizza", "oversized ham disc", "greyshirt's bane", \
+	 "OP's mom")
```
Comment (Manly-Dwarf) [inline hippiestation/code/modules/antagonists/changeling/horrorform.dm:26]: Nice.

**Review by Manly-Dwarf — DISMISSED** [review]
Don't think it's necessary to remove the sprites as well, but code is good... I guess?
Hilarious that you're leaving free horror form in, re-adding adrenals, AND reverting all of the nerfs. The fire rises.

**Comment by biggiOS** [comment]
+1 kbb is getting salty :pepescheming:

**Comment by Manly-Dwarf** [comment]
No, I actually told Amari to revert the PR. Re-adding revive isn't gonna fix anything. besides, ling being super OP was at least fun for the ling.
I'm not satisfied with my nerfs because they only made ling more boring. While it's true that ling was terribly designed and needed the nerfs, that's not an excuse to make the antag as a whole unfun for everyone.

Also I want to get the gamemode removed, and pissing off the Head Admins in-game is the best way to do that. Now, if only I could get them to actually play...

File: hippiestation/code/modules/antagonists/changeling/powers/adrenals.dm [hunk 3]
```diff
@@ -0,0 +1,33 @@
+#define ADRENAL_COOLDOWN 5 MINUTES
```
Comment (Manly-Dwarf) [inline hippiestation/code/modules/antagonists/changeling/powers/adrenals.dm]: Has this been tested? I don't mess around with world time, so I don't know if "5 MINUTES" will do the trick.

File: hippiestation/code/modules/antagonists/changeling/powers/adrenals.dm [hunk 4]
```diff
@@ -0,0 +1,33 @@
...
+	req_stat = UNCONSCIOUS
+	var/last_run = 0
+
+//Recover from stuns.
+/obj/effect/proc_holder/changeling/adrenaline/sting_action(mob/living/user)
+	if(user.on_fire)
+		to_chat(user, "<span class='Hey stupid, you're on fire.</span>")
+		return FALSE
+	if(world.time >= last_run + ADRENAL_COOLDOWN)
+		to_chat(user, "<span class='Hey stupid, you can only use adrenals every 5 minutes.</span>")
```
Comment (Manly-Dwarf) [inline hippiestation/code/modules/antagonists/changeling/powers/adrenals.dm]: I appreciate keeping the sass from my ling nerfs.

File: hippiestation/code/modules/antagonists/changeling/powers/adrenals.dm [hunk 5]
```diff
@@ -0,0 +1,33 @@
+#define ADRENAL_COOLDOWN 5 MINUTES
+
+/obj/effect/proc_holder/changeling/adrenaline
+	name = "Adrenaline Sacs"
+	desc = "We evolve additional sacs of adrenaline throughout our body."
+	helptext = "Removes all stuns instantly and adds a short-term reduction in further stuns. Can be used while unconscious. Continued use poisons the body."
```
Comment (Manly-Dwarf) [inline hippiestation/code/modules/antagonists/changeling/powers/adrenals.dm]: I don't see poison anywhere around here.

**Review by Manly-Dwarf — DISMISSED** [review]

File: hippiestation/code/modules/antagonists/changeling/powers/adrenals.dm (hunk 3 above)
Comment (ghost) [reply hippiestation/code/modules/antagonists/changeling/powers/adrenals.dm]: `MINUTES` is a define

Comment (steamp0rt) [reply hippiestation/code/modules/antagonists/changeling/powers/adrenals.dm]: `changelingadrenaline` and `changelinghaste` cause tox damage when overdosed

https://github.com/HippieStation/HippieStation/blob/8f6562ee869ee77ddb4688558500f1f6264d860a/code/modules/reagents/chemistry/reagents/medicine_reagents.dm#L1125-L1162

File: hippiestation/code/modules/antagonists/changeling/powers/adrenals.dm (hunk 5 above)
Comment (ghost) [reply hippiestation/code/modules/antagonists/changeling/powers/adrenals.dm]: `changelingadrenaline` and `changelinghaste` cause tox damage when overdosed

https://github.com/HippieStation/HippieStation/blob/8f6562ee869ee77ddb4688558500f1f6264d860a/code/modules/reagents/chemistry/reagents/medicine_reagents.dm#L1125-L1162

File: hippiestation/code/modules/antagonists/changeling/powers/adrenals.dm (hunk 3 above)
Comment (Manly-Dwarf) [reply hippiestation/code/modules/antagonists/changeling/powers/adrenals.dm]: ok thanks
picardia2

**Comment by Lxtoofonea** [comment]
+1, this needed to be done for a long time

**Comment by yoyobatty** [comment]
Nice branch name autist

File: hippiestation.dme [hunk 6]
```diff
@@ -3004,15 +3003,9 @@
...
-#include "hippiestation\code\modules\antagonists\changeling\absorb.dm"
-#include "hippiestation\code\modules\antagonists\changeling\changeling.dm"
-#include "hippiestation\code\modules\antagonists\changeling\fakedeath.dm"
-#include "hippiestation\code\modules\antagonists\changeling\glands.dm"
-#include "hippiestation\code\modules\antagonists\changeling\headcrab.dm"
 #include "hippiestation\code\modules\antagonists\changeling\horrorform.dm"
-#include "hippiestation\code\modules\antagonists\changeling\lesserform.dm"
-#include "hippiestation\code\modules\antagonists\changeling\mutations.dm"
-#include "hippiestation\code\modules\antagonists\changeling\revive.dm"
+#include "hippiestation\code\modules\antagonists\changeling\powers\adrenals.dm"
```
Comment (GuyonBroadway) [inline hippiestation.dme]: First of all HOLY FUCK NO! NO ANTISTUNS!!

secondly don't remake the file you doughnut hole, just re-include the tg file.

**Review by GuyonBroadway — DISMISSED** [review]

**Comment by Carbonhell** [comment]
@lxtoofonea
This readds adrenals too, are you sure about your +1

**Comment by DaydreamPuppeh** [comment]
Fuck yeah. Fart sting

**Comment by WarRage333** [comment]
Make it you can only revive after you absorb another changeling

**Comment by ghost** [comment]
^ best idea. 
charge based revification, no more doing it infinitely. work for your 1up

**Comment by ghost** [comment]
Remove the fucking adrenals, or change their functionality to a stun reducer. Do I need to say that antistuns are fucking bad?

**Comment by ghost** [comment]
@HideAndSeekLOGIC I nerfed em'. Made em,
A. not work while on fire, 
B. have hardcoded 5 minute cooldown,
C. take over half of the chems.

**Comment by ghost** [comment]
@steamp0rt fucking nice, that's some good fucking shit

File: hippiestation.dme (hunk 6 above)
Comment (ghost) [reply hippiestation.dme]: i changed a good part of the function lol

**Comment by Manly-Dwarf** [comment]
It's going to be quite useful getting everyone to fart their asses off, so they can't superfart you.
Dang, Amari, you're better at design for ling than you are at vampire and you don't even realize it.

**Comment by Eagle-Eyess** [comment]
-1 everything is fine except for the anti stuns.... remove that and you got my +1

**Comment by ghost** [comment]
@Eagle-Eyess lol can you not win a fight without stuns

**Comment by Carbonhell** [comment]
@steamp0rt lol can you not win a fight without antistuns

**Comment by ghost** [comment]
@Carbonhell na I prefer tossing hellfoam and GTFOing

**Comment by ghost** [comment]
oh nice 
capture

**Comment by Manly-Dwarf** [comment]
YES
THIS IS MURDERBONE TAKEN
TO THE **MAX**

**Comment by JohnMcLaime** [comment]
Nice, those ling nerfs would have been better if they removed the gamemode outright in favor of traitor, the abilities are boring, literally a glock gun that fires off your gene/ling mana thingie or a taser arm? Traitor has more options - more fun than this, even the meme ability it had - changing someone identity, wasnt buffed, it still takes up ridiculous ammount of ling spec points and eats 50 mana, thats shit for an ability that doesnt really create confusion since people still keep their clothes and ID to look distinctly from you. Bbbut you can strip them and take their shit away, that way people wont see a difference, you say? If you can do that, why not just kill them and take their identity in first place.

**Comment by bardofrage** [comment]
If adrenals are removed, i'll +1 them. As is (with adrenals). -1

**Comment by ghost** [comment]
@bardofrage done

**Comment by Pyko1** [comment]
Dont you start taking damage if you dont have butt and you are injected with fartium

**Review by yoyobatty — APPROVED** [review]

**Comment by earthmoonjune** [comment]
sure, +1 so long as no adrenals.

**Comment by ghost** [comment]
this just got merged ya know

**Comment by k-k-karma** [comment]
no it didn't

**Pull request merged**